src/Microsoft.Azure.SignalR.Common/Endpoints/AccessKeyForMicrosoftEntra.cs

```diff
@@ -37,12 +35,14 @@ internal class AccessKeyForMicrosoftEntra : AccessKey
 
         private static readonly TimeSpan GetAccessKeyIntervalWhenUnauthorized = TimeSpan.FromMinutes(5);
 
-        private static readonly TimeSpan GetAccessKeyRetryInterval = TimeSpan.FromSeconds(GetAccessKeyRetryIntervalInSec);
+        private static readonly TimeSpan GetAccessKeyRetryInterval = TimeSpan.FromSeconds(3);
```

Comment:
why not using const?

Reply:
This value hasn't been used elsewhere.

And I think `private static readonly` is already a sort of `const`?